docs/features/search/collators.md

```diff
@@ -0,0 +1,107 @@
+---
+id: collators
+title: Collators
+description: Indexing you Backstage content with Collators
+---
+
+Backstage includes 2 [collators](./concepts.md#collators) out of the box for the [Catalog](#catalog) and [TechDocs](#techdocs). There's also some from the [Backstage Community](#community-collators) too!
```

Comment:
@emmaindal, I've removed the line from this opening section that was already in the Concepts page and linked to that section. Does that seem better to you?